drivers/led/lp5562.c

```diff
@@ -68,12 +68,12 @@ LOG_MODULE_REGISTER(lp5562);
  * therefore (16 * 63) = 1008ms. We round it down to 1000ms to be on the safe
  * side.
  */
-#define LP5562_MAX_BLINK_PERIOD K_MSEC(1000)
```

Comment:
Huh, I was surprised to see these K_MSEC() removed in this C file, were they being applied twice?

Reply:
More likely they were being used as terms in an expression (like "take half the blink period").  In general I tried to choose changes to minimize the size of the patch.  So if you had one timeout defined as a number but used unmodified in several of places, I'd put the macro in the declaration.  If it was a time value used with modification, I'd leave it as an integer in the header and change the usage to k_msleep(), etc...